tests/ansible_collections/roles/remove-rcmtools/tasks/clean_up_el7.yml

```
@@ -0,0 +1,6 @@
+---
+- name: Remove rcmtools packages installed on the system
+  # We cannot use yum remove as it also removes dependency packages (like even c2r) that we need to keep installed
+  ansible.builtin.shell: yum list installed | grep @rcmtools | awk '{print $1}' | xargs rpm -e --nodeps
+- name: Remove rcmtools repo from the system
+  ansible.builtin.shell: rm -f /etc/yum.repos.d/rcmtools.repo
```

```suggestion
  ansible.builtin.yum_repository:
    file: rcmtools
    name: rcmtools
    state: absent

- name: Clean yum metadata
  ansible.builtin.command: yum clean metadata
```
Can we rather go with the [built-in yum_repository module](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/yum_repository_module.html) ?
Ideally, clean the metadata as well.

---

ok, I'll change it